### PR TITLE
Better association homepage

### PR DESCRIPTION
**Pull request by abocquet**

### CODE REVIEW

File: frontend/src/components/associations/Home.tsx [hunk 1]
```diff
@@ -2,16 +2,42 @@ import React from "react";
...
-            <PageTitle>{association.name}</PageTitle>
-            <p>Home page !!!</p>
+    return status === "loading" ? (
+        <Loading />
+    ) : status === "error" ? (
+        <ErrorMessage>{error}</ErrorMessage>
+    ) : data.count === 0 ? (
+        <Container className={"text-center"}>
+            <p style={{ fontSize: "10em" }} className={"m-0"}>
+                🏡
```
Comment (Piwakk) [inline frontend/src/components/associations/Home.tsx:26]: This triggers a warning:  Emojis should be wrapped in `span`, have role="img", and have an accessible description with aria-label or aria-labelledby

File: frontend/src/components/associations/Home.tsx [hunk 2]
```diff
@@ -2,16 +2,42 @@ import React from "react";
...
+            </p>
+            <h1>
+                Accueil <br />
+                <br />
+                <small className="text-muted">
+                    Pour définir une page d'accueil, créez une page appellée
+                    "Accueil". C'est aussi simple que ça !
```
Comment (Piwakk) [inline frontend/src/components/associations/Home.tsx:33]:
```suggestion
                    « Accueil ». C'est aussi simple que ça !
```

File: frontend/src/components/associations/Home.tsx [hunk 3]
```diff
@@ -2,16 +2,42 @@ import React from "react";
 import { PageTitle } from "../utils/PageTitle";
 import Container from "react-bootstrap/Container";
 import { Loading } from "../utils/Loading";
+import { api, useBetterPaginatedQuery } from "../../services/apiService";
+import { ErrorMessage } from "../utils/ErrorPage";
+import { AssociationPageRenderer } from "./page/Show";
 
 export const AssociationHome = ({ association }) => {
```
Comment (Piwakk) [inline frontend/src/components/associations/Home.tsx:9]: I have a TypeError: association.myRole.permissions is undefined when going to this page logged in as `19simple`.

File: frontend/src/components/associations/medias/List.tsx [hunk 4]
```diff
@@ -73,6 +74,21 @@ export const AssociationFilesystemList = ({ association }) => {
...
+                                    title={"Gestion des médias"}
+                                    emoji={"🗂️"}
+                                    emojiAriaLabel="Des fiches cartonées"
+                                >
+                                    Aucun fichier pour l'instant.
+                                    {association.myRole.permissions?.includes(
+                                        "media"
+                                    )
+                                        ? "Pour ajouter des fichiers, cliquez sur le bouton 'Ajouter des fichiers'."
+                                        : "Revenez quand les responsables de l'association en auront ajouté !"}
```
[the remark this answers is not in the document]
Comment (abocquet) [reply frontend/src/components/associations/medias/List.tsx:89]: http://www.aidenet.eu/conjugaison44.htm

File: frontend/src/components/associations/Home.tsx [hunk 5]
```diff
@@ -27,8 +28,19 @@ export const AssociationHome = ({ association }) => {
...
+                        Pour définir une page d'accueil,{" "}
+                        <Link
+                            to={`/associations/${association.id}/pages/nouvelle`}
+                        >
+                            créez une page
+                        </Link>{" "}
+                        appelée « Accueil ». C'est aussi simple que ça !
+                    </>
+                ) : (
+                    "Rien à voir ici pour le moment mais  vous pouvez regarder les différentes rubriques dans la barre de navigation à gauche."
```
Comment (Piwakk) [inline frontend/src/components/associations/Home.tsx:42]:
```suggestion
                    "Rien à voir ici pour le moment mais vous pouvez accéder aux différentes rubriques grâce à la barre de navigation à gauche."
```

**Pull request merged**